### PR TITLE
Use templates

### PR DESCRIPTION
**Pull request by sanderson**
Instructions for how to use InfluxDB templates.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/influxdb-templates/create.md [hunk 1]
```diff
@@ -45,6 +45,11 @@ In **InfluxDB Cloud**, your user account is an organization.
 
 4. Export the template _(see [below](#export-a-template))_.
 
+{{% warn %}}
+InfluxDB templates do not support the [table visualization type](/v2.0/visualize-data/visualization-types/table/).
+Dashboard cells that use the table visualization are not included in exported templates.
```
Comment (pierwill) [inline content/v2.0/influxdb-templates/create.md]: ~the~ table visualization

File: content/v2.0/influxdb-templates/use.md [hunk 2]
```diff
@@ -1,20 +1,193 @@
...
+
+- [View a template summary](#view-a-template-summary)
+- [Validate a template](#validate-a-template)
+- [Install templates](#install-templates)
+- [Use InfluxDB community templates](#use-influxdb-community-templates)
+
+## View a template summary
+To view a summary of what's included in a template before installing the template,
+use the [`influx pkg summary` command](/v2.0/reference/cli/influx/pkg/summary/).
+Summarize templates stored in your local filesystem or from a URL.
```
Comment (pierwill) [inline content/v2.0/influxdb-templates/use.md]: consider removing line 25.

File: content/v2.0/influxdb-templates/use.md [hunk 3]
```diff
@@ -1,20 +1,193 @@
...
+  --secret=BAZ=quz
+```
+
+_To add a secret after installing a template, see [Add secrets](/v2.0/security/secrets/manage-secrets/add/)._
+
+## Use InfluxDB community templates
+The [InfluxDB Community Templates repository](https://github.com/influxdata/community-templates/)
+is home to a growing number of InfluxDB templates developed and maintained by
+others in the InfluxData community.
+Install community templates directly from GitHub using a template's raw code URL
```
Comment (pierwill) [inline content/v2.0/influxdb-templates/use.md]:
```
[...] raw code URL
(by clicking the **Raw** button on the `.yml` file's GitHub page)
[...]
```

File: content/v2.0/influxdb-templates/create.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ menu:
     parent: InfluxDB templates
     name: Create a template
     identifier: Create an InfluxDB template
-weight: 101
+weight: 102
 v2.0/tags: [templates]
 ---
```
Comment (kelseiv) [inline content/v2.0/influxdb-templates/create.md]: To create an InfluxDB template, do one of the following:
- [Import a template](...)
- [Create a template from an existing dashboard](...)
- [Clone a template](...)

File: content/v2.0/influxdb-templates/use.md [hunk 5]
```diff
@@ -1,20 +1,193 @@
...
-  - View package summaries before installing
-  - Validate a package before installing
-  - Use a template from a remote URL  
-  - Define secrets while installing a template
+Use the `influx pkg` command to summarize, validate, and install templates from
+your local filesystem and from URLs.
+
+- [View a template summary](#view-a-template-summary)
+- [Validate a template](#validate-a-template)
+- [Install templates](#install-templates)
```
Comment (kelseiv) [inline content/v2.0/influxdb-templates/use.md]: Install a template
Use an InfluxDB community template

File: content/v2.0/influxdb-templates/use.md (hunk 2 above)
Comment (kelseiv) [inline content/v2.0/influxdb-templates/use.md]: View a summary of a template stored in...

File: content/v2.0/influxdb-templates/use.md [hunk 6]
```diff
@@ -1,20 +1,193 @@
...
+influx pkg summary -u <template-url>
+
+# Example
+influx pkg summary -u https://raw.githubusercontent.com/influxdata/community-templates/master/linux_system/linux_system.yml
+```
+{{% /code-tab-content %}}
+{{% /code-tabs-wrapper %}}
+
+## Validate a template
+To validate a template before installing it or for troubleshooting purposes, use
```
Comment (kelseiv) [inline content/v2.0/influxdb-templates/use.md]: To validate a template before you install it or troubleshoot an existing template

File: content/v2.0/influxdb-templates/use.md [hunk 7]
```diff
@@ -1,20 +1,193 @@
...
+the [`influx pkg validate` command](/v2.0/reference/cli/influx/pkg/validate/).
+Validate templates stored in your local filesystem or from a URL.
```
Comment (kelseiv) [inline content/v2.0/influxdb-templates/use.md]: Validate a template stored...

File: content/v2.0/influxdb-templates/use.md [hunk 8]
```diff
@@ -1,20 +1,193 @@
...
+# Syntax
+influx pkg validate -u <template-url>
+
+# Example
+influx pkg validate -u https://raw.githubusercontent.com/influxdata/community-templates/master/linux_system/linux_system.yml
+```
+{{% /code-tab-content %}}
+{{% /code-tabs-wrapper %}}
+
+## Install templates
```
Comment (kelseiv) [inline content/v2.0/influxdb-templates/use.md]: Install a template

File: content/v2.0/influxdb-templates/use.md [hunk 9]
```diff
@@ -1,20 +1,193 @@
...
+Use the [`influx pkg` command](/v2.0/reference/cli/influx/pkg/) to install templates
+from your local filesystem or from URLs.
```
Comment (kelseiv) [inline content/v2.0/influxdb-templates/use.md]: to install a template from...

File: content/v2.0/influxdb-templates/use.md [hunk 10]
```diff
@@ -1,20 +1,193 @@
...
+
+- [Install templates from files](#install-templates-from-files)
+- [Install templates from URLs](#install-templates-from-urls)
+- [Install templates from both files and URLs](#install-templates-from-both-files-and-urls)
+- [Include secrets when installing a template](#include-secrets-when-installing-a-template)
```
Comment (kelseiv) [inline content/v2.0/influxdb-templates/use.md]: keep singular for consistency

File: content/v2.0/influxdb-templates/use.md [hunk 11]
```diff
@@ -1,20 +1,193 @@
...
+# Install a single template
+influx pkg -f /path/to/template.yml
+
+# Install multiple templates
+influx pkg \
+  -f /path/to/this/template.yml \
+  -f /path/to/that/template.yml
+```
+
+#### Install all templates in a directory
```
Comment (kelseiv) [inline content/v2.0/influxdb-templates/use.md]: maybe break out how to install multiple templates in a directory into it's own section?

File: content/v2.0/influxdb-templates/use.md (hunk 3 above)
Comment (kelseiv) [inline content/v2.0/influxdb-templates/use.md]: the template's remote URL? --how we're wording it on the github repo, so either way, we'll want to keep consistent terminology

File: content/v2.0/influxdb-templates/use.md [hunk 12]
```diff
@@ -1,20 +1,193 @@
...
+```
+
+_To add a secret after installing a template, see [Add secrets](/v2.0/security/secrets/manage-secrets/add/)._
+
+## Use InfluxDB community templates
+The [InfluxDB Community Templates repository](https://github.com/influxdata/community-templates/)
+is home to a growing number of InfluxDB templates developed and maintained by
+others in the InfluxData community.
+Install community templates directly from GitHub using a template's raw code URL
+or download them to your local machine.
```
Comment (kelseiv) [inline content/v2.0/influxdb-templates/use.md]: download a template

File: content/v2.0/influxdb-templates/use.md (hunk 8 above)
Comment (sanderson) [reply content/v2.0/influxdb-templates/use.md]: I'm going to leave this as plural. The instructions walk through installing both single templates and multiple templates at once.

File: content/v2.0/influxdb-templates/create.md (hunk 4 above)
Comment (sanderson) [reply content/v2.0/influxdb-templates/create.md]: These items don't apply to the template creation workflow.

**Pull request merged**